components/function-runtimes/python39/kubeless/tracing.py

```diff
@@ -56,13 +56,10 @@ def _get_tracer(jaeger_endpoint: str, service_name: str) -> trace.Tracer:
     return trace.get_tracer(__name__)
 
 
-def _is_jaeger_available(jaegerEndpoint) -> bool:
+def _is_tracecollector_available(tracecollectorEndpoint) -> bool:
     try:
-        res = requests.get(jaegerEndpoint, timeout=2)
+        res = requests.get(tracecollectorEndpoint, timeout=2)
 
-        # 405 is the right status code for the GET method if jaeger service exists 
-        # because the only allowed method is POST and usage of other methods are not allowe
-        # https://github.com/jaegertracing/jaeger/blob/7872d1b07439c3f2d316065b1fd53e885b26a66f/cmd/collector/app/handler/http_handler.go#L60
```

Comment:
Why did we decide to remove this comment? 🤔 It describes why we expect `405` as the right status code

Reply:
added it back